src/main/webapp/app/entities/submission/submission.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { SERVER_API_URL } from 'app/app.constants';
+
+import * as moment from 'moment';
+
+import { createRequestOption } from 'app/shared';
+import { Result } from 'app/entities/result';
+import { Submission } from 'app/entities/submission';
+import { Participation } from 'app/entities/participation';
+
+export type EntityResponseType = HttpResponse<Submission>;
+export type EntityArrayResponseType = HttpResponse<Submission[]>;
+
+@Injectable({ providedIn: 'root' })
+export class SubmissionService {
+    public resourceUrl = SERVER_API_URL + 'api/submissions';
+    public resourceUrlParticipation = SERVER_API_URL + 'api/participation';
+
+    constructor(private http: HttpClient) {}
+
+    find(submissionId: number): Observable<EntityResponseType> {
```

Comment:
Do we really need this method here and find one particular submission?

Reply:
No, we are not using this method. I removed the method.